specification/ai/Azure.AI.Projects/tspconfig.yaml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  "service-dir":
+    default: "sdk/ai"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "data-plane"
+    emit-lro-options: "none"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/azure-ai-projects-1dp.json"
```

Comment:
it's time to get rid of the "-1dp" here...

Reply:
I have a PR out to change this: https://github.com/Azure/azure-rest-api-specs/pull/34333